public/app/plugins/panel/canvas/Connections.tsx

```diff
@@ -89,6 +90,15 @@ export class Connections {
   };
 
   handleMouseLeave = (event: React.MouseEvent | React.FocusEvent) => {
+    if (event.relatedTarget instanceof HTMLImageElement) {
+      const relatedTarget = event.relatedTarget;
+      assertInstanceOf(relatedTarget, HTMLImageElement);
+      // If mouse is leaving INTO the anchor image, don't remove div
+      if (relatedTarget.getAttribute('alt') === connectionAnchorAlt) {
+        return;
+      }
+    }
```

Comment:
`assertInstanceOf` check seems a little redundant, maybe we can do something along these lines?
```suggestion
    // If mouse is leaving INTO the anchor image, don't remove div
    if (event.relatedTarget instanceof HTMLImageElement && event.relatedTarget.getAttribute('alt') === connectionAnchorAlt) {
      return;
    }
```

Reply:
That's what I originally tried, but you cannot access getAttribute unless you assert the type. If you use `const relatedTarget = event.relatedTarget as HTMLImageElement;` betterer will error "Do not use any type assertions."

Reply:
Hm locally the above approach works for me with no issues 🤔 TS should know that relatedTarget will have an 'alt' attribute given it is checked first to be an instance of a html image element

Reply:
Good catch, that worked for me!